src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -319,50 +368,116 @@ public WPI_Pigeon2 getGyro() {
+                  if (isFirstPath) {
+                    this.resetOdometry(traj.getInitialHolonomicPose());
+                  }
+                }),
+            new PPSwerveControllerCommand(
+                traj,
+                this::getPose, // Pose supplier
+                Constants.SwerveDrive.DRIVE_KINEMATICS,
+                Cal.SwerveSubsystem.PATH_X_CONTROLLER,
+                Cal.SwerveSubsystem.PATH_Y_CONTROLLER, // usually the same values as X controller
+                Cal.SwerveSubsystem.PATH_THETA_CONTROLLER,
+                this::setModuleStates, // Module states consumer
+                false, // Should the path be automatically mirrored depending on alliance color.
+                this // Requires this drive subsystem
+                ),
+            new InstantCommand(
                 () -> {
-                  return !isFirstPath;
+                  targetHeadingDegrees = getHeadingDegrees();
                 }),
-        new InstantCommand(
-            () -> {
-              // Reset odometry for the first path you run during auto
-              if (isFirstPath) {
-                this.resetOdometry(traj.getInitialHolonomicPose());
-              }
-            }),
-        new PPSwerveControllerCommand(
-            traj,
-            this::getPose, // Pose supplier
-            Constants.SwerveDrive.DRIVE_KINEMATICS, // SwerveDriveKinematics
-            Cal.SwerveSubsystem
-                .PATH_X_CONTROLLER, // X controller. Tune these values for your robot. Leaving them
-            // 0 will only use feedforwards.
-            Cal.SwerveSubsystem
-                .PATH_Y_CONTROLLER, // Y controller (usually the same values as X controller)
-            Cal.SwerveSubsystem
-                .PATH_THETA_CONTROLLER, // Rotation controller. Tune these values for your robot.
-            // Leaving them 0 will only use feedforwards.
-            this::setModuleStates, // Module states consumer
-            true, // Should the path be automatically mirrored depending on alliance color.
-            // Optional, defaults to true
-            this // Requires this drive subsystem
-            ),
-        new InstantCommand(
-            () -> {
-              targetHeadingDegrees = getPose().getRotation().getDegrees();
-            }));
+            new PrintCommand("Finished a trajectory"))
+        .withName("Follow trajectory");
+  }
+
+  public void setLimelightTargetFromTransform(Transform2d transform, double latencySec) {
+    // Transform is to get the limelight to the correct location, not to get the robot
+    // Here we correct for that
+    Transform2d flipTransform =
+        new Transform2d(
+            new Translation2d(-transform.getX(), transform.getY()), transform.getRotation());
+    System.out.println("Flip Transform: " + flipTransform.getX() + " " + flipTransform.getY());
+
+    Pose2d curPose = getPose();
+    latencySec = latencySec + 0.05;
```

Comment:
0.05 should be a variable

Reply:
Also can do += which is *slightly* more concise

Reply:
Sure why not